doc/user-guide.md

```diff
@@ -106,6 +106,28 @@ After modifying the `*_types.go` file always run the following command to update
 $ operator-sdk generate k8s
 ```
 
+To update the Open API validation section in the CRD `deploy/crds/cache_v1alpha1_memcached_crd.yaml`, run the following command. 
```

Comment:
I'd like to see this highlighted as a subsection so it's easier to point to. Also `Open API`=>`OpenAPI`.
```suggestion
### OpenAPI validation
To update the OpenAPI validation section in the CRD `deploy/crds/cache_v1alpha1_memcached_crd.yaml`, run the following command. 
```